app/assets/javascripts/discourse/app/components/bread-crumbs.hbs

```diff
@@ -17,17 +17,7 @@
   {{#if breadcrumb.hasOptions}}
     <li
       style={{html-safe
-        (if
-          breadcrumb.category
-          (concat
-            "--category-bg-color:#"
-            breadcrumb.category.color
-            "; "
-            "--category-text-color:#"
-            breadcrumb.category.text_color
-            ";"
-          )
-        )
+        (if breadcrumb.category (category-variables breadcrumb.category))
```

Comment:
`categoryVariables`  function already returns html-safe string

Reply:
Also, if it's used only in this component it doesn't need to be a global helper, you can add a helper method to the component and use it like e.g. `{{this.categoryStyle breadcrumb.category}}`

Reply:
I am planning on using it in a following PR.